library/linear.lib

```diff
@@ -1564,59 +1699,16 @@ $FPLIST
  Texas*DSBGA*Pitch0.5mm*
 $ENDFPLIST
 DRAW
-X GND A3 0 -400 100 U 50 50 0 0 W
-X VCC A2 0 400 100 D 50 50 0 0 W
 S -350 -300 350 300 0 1 10 f
 S -350 -300 350 300 0 1 10 f
+X VCC A2 0 400 100 D 50 50 0 0 W
+X GND A3 0 -400 100 U 50 50 0 0 W
+X DIS A1 500 0 150 L 50 50 1 1 I
+X THR B1 500 -200 150 L 50 50 1 1 I
+X CV C1 -500 0 150 R 50 50 1 1 I
+X R C2 -500 -200 150 R 50 50 1 1 I I
 X TR B3 -500 200 150 R 50 50 1 1 I
 X Q C3 500 200 150 L 50 50 1 1 O
-X R C2 -500 -200 150 R 50 50 1 1 I I
-X CV C1 -500 0 150 R 50 50 1 1 I
-X THR B1 500 -200 150 L 50 50 1 1 I
-X DIS A1 500 0 150 L 50 50 1 1 I
-ENDDRAW
-ENDDEF
-#
-# LMV321
-#
-DEF LMV321 U 0 20 Y Y 1 F N
-F0 "U" 0 200 50 H V L CNN
-F1 "LMV321" 0 -200 50 H V L CNN
-F2 "" 0 0 50 H I L CNN
-F3 "" 0 0 50 H I C CNN
-ALIAS MCP6561U MCP6001U OPA333xxDCK OPA376xxDCK
-$FPLIST
- SOT*23*
- *SC*70*
-$ENDFPLIST
-DRAW
-P 4 0 1 10 -200 200 200 0 -200 -200 -200 200 f
-X V- 2 -100 -300 150 U 50 50 0 1 W
-X V+ 5 -100 300 150 D 50 50 0 1 W
-X + 1 -300 100 100 R 50 50 1 1 I
-X - 3 -300 -100 100 R 50 50 1 1 I
-X ~ 4 300 0 100 L 50 50 1 1 O
-ENDDRAW
-ENDDEF
-#
-# LMV331
```

Comment:
LMV331 deleted.
LMV331M7 is now an alias of Amp.S.SC-70-5.Ni-Pn-Ii-O-Pp
LMV331M5 is now an alias of Amp.S.SOT-23-5.Ni-Pn-Ii-O-Pp